### DISCUSSION TITLE
BEQDesigner and ezBEQ on QNAP NAS as QVM

### DISCUSSION THREAD

**Discussion by d45p00k13** [3ll3d00d/beqdesigner#455] [Q&A] [answered]
Dear Community,

in order to use BEQDesigner on the fly, I wanted to install ezBEQ additionally on my new QNAP NAS.
This combination already works on my desktop completely.

I am stuck right at the start: I installed virtualization station as app on the NAS, configured an win11 VM, installed it and the OS itself runs flawlessly. I started the exe from BEQDesigner as pre-requiste for my task but the console / cmd-line opens and after a while it closes w/o any message and it is obviously not installed.

Can you help me, to get it running? A hint would be helpful if there are any other requirements for an installation and use to be fulfilled by the QVM.

Thanks in advance,
D45p00k13

**Comment by 3ll3d00d** [comment]
Try running the exe from a CMD window instead, I expect you will see some error messages before it exits.

**Comment by d45p00k13** [comment]
Thanks for your fast response! You were right, here are the infos:

.\beqdesigner_10.exe
Traceback (most recent call last):
  File "app.py", line 10, in <module>
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "matplotlib\__init__.py", line 259, in <module>
  File "matplotlib\__init__.py", line 253, in _check_versions
  File "importlib\__init__.py", line 127, in import_module
  File "PyInstaller\loader\pyimod02_importers.py", line 352, in exec_module
  File "kiwisolver\__init__.py", line 8, in <module>
ImportError: DLL load failed while importing _cext: Das angegebene Modul wurde nicht gefunden.
[7996] Failed to execute script 'app' due to unhandled exception!

I installed latest x64 package for python. python-3.12.0a7-amd64

**Comment by 3ll3d00d** [comment, marked as answer]
it will most likely be a missing one of these https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

there must be some incompatibility between your VM and how it is built, you can try manually installing one of the vc redist packages and run it again

you don't need to install python itself btw

**Comment by d45p00k13** [comment]
Hoi!
Thanks for your response! It took me a while to re-create the QVM since it crashed while migrating the raid... na^^ besides that, it was the solution to install the latest VC-redist. BEQ Designer starts and seems to do its work. Thanks for your effort and interest! I will go on and setup the rest. Thanks again!
